cairo/src/word.rs

```diff
@@ -0,0 +1,429 @@
+//! The Cairo language works natively for field elements in the finite field with
+//! modulus 0x800000000000011000000000000000000000000000000000000000000000001
```

Comment:
This is the size of the finite field that they use in Cairo. It comes from the operation: 2 ^ 251 + 17 * 2 ^ 192 + 1. 

This means that this number `0x800000000000011000000000000000000000000000000000000000000000001` would become zero in a real Cairo execution, using the Cairo implementation. In our case however, it would actually correspond to the full amount `0x800000000000011000000000000000000000000000000000000000000000001`, as it fits inside the Pallas fields.

Reply:
maybe include a link to some source stating that?

Reply:
Awesome, I will :)